source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceSetView.swift

```diff
@@ -81,15 +87,7 @@ class ACRChoiceSetView: NSView, InputHandlingViewProtocol {
     }
     
     var isValid: Bool {
-        return true
-    }
-    
-    var isRequired: Bool {
-        return false
-    }
-    
-    func showError() {
-        // do nothing
+        return isRequired ? stackview.arrangedSubviews.contains(where: { let elem = $0 as? ACRChoiceButton; return elem?.state == .on }) : true
     }
```

Comment:
Not sure, do you need `isBasicValidationSatidfied` here?

Reply:
No Akaash, in choiceset like we discussed, people can make the value empty string and for input.toggle it either submits true/falssse, never empty